it/uk/gov/hmrc/agentsubscriptionfrontend/controllers/BusinessIdentificationControllerISpec.scala

```diff
@@ -667,13 +667,13 @@ trait CheckAgencyControllerISpec extends BaseISpec with SessionDataMissingSpec {
           Some(businessAddress), Some("someone@example.com")))
```

Comment:
Trivial: Back up on line 657 "redirect to confirm your agency" is now "redirect to confirm your business"

Reply:
Same on line 637